api/institutions/views.py

```diff
@@ -233,5 +240,89 @@ class InstitutionRegistrationList(InstitutionNodeList):
     def get_queryset(self):
         inst = self.get_institution()
         query = self.get_query_from_request()
-        nodes = list(Node.find_by_institution(inst, query))
+        nodes = list(Node.find_by_institutions(inst, query))
         return [node for node in nodes if not node.is_retracted]
```

Comment:
This could be slow -- you're enumerating this queryset twice. ~~Why not just `return Node.find_by_institutions(inst, query & Q('is_retracted', 'eq', False))`?~~ Because `is_retracted` is a property. Still, you shouldn't need to `list` the queryset the first time.